docs/src/modules/components/AppFrame.js

```diff
@@ -141,6 +142,16 @@ const styles = (theme) => ({
       display: 'none',
     },
   },
+  banner: {
+    display: 'block',
+    padding: 4,
```

Comment:
I'd a little bit more padding, 8 here. Just to go along with the smaller font size.